src/libopensc/pkcs15.c

```diff
@@ -856,6 +856,19 @@ sc_pkcs15_card_clear(struct sc_pkcs15_card *p15card)
 }
 
 
+int sc_copy_gost_params(struct sc_pkcs15_gost_parameters *dst, struct sc_pkcs15_gost_parameters *src)
```

Comment:
Why don't you move this implementation to `src/pkcs15init/pkcs15-lib.c`, which is the only place where this is used?